instagram/src/App.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import './App.css';
+import dummyData from './dummy-data'
+import PostContainer from './components/PostContainer/PostContainer'
+import SearchBar from './components/SearchBar/SearchBar'
+import Authenticate from './components/Authentication/Authenticate'
+
+class App extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      dummyDataList: [],
+      inputText:''
+    }
+  }
+
+
+    componentDidMount() {
+      this.setState({ dummyDataList: dummyData })
+    }
+
+    search = (term) => {
+      let dummyDataList = dummyData.filter(post  => post.username.includes(term))
+      this.setState({ dummyDataList })
+    }
+
+
+  render() {
+    return (
+      <div className="App">
+      <header className="search-bar">
+                <SearchBar search={this.search}/>
+            </header> 
+
+             <React.Fragment className="App-renderer">
```

Comment:
You cannot add a class name to a Fragment. Fragments just exist to help group child elements, but they don't actually render any element to the page. If you need something to have a class, use a div.

Reply:
Got it! As I understand that React>fragment is used so as to not add any new nodes into your DOM.